tests/dummy/mirage/config.js

```diff
@@ -6,7 +6,7 @@ export default function() {
   this.post("/realms/test-realm/protocol/openid-connect/token", function() {
     let tokenBody = btoa(
       JSON.stringify({
-        exp: Math.round(new Date().getTime() + (30 * 60 * 1000) / 1000)
+        exp: Math.round(new Date().getTime() + 30 * 60)
```

Comment:
what is this actually used for? Still needed after rewrite of `expires_in`?

Reply:
No, this is no longer needed and therefore removed